auto_round/compressors/base.py

```diff
@@ -2980,10 +2980,16 @@ def _quantize_block(
         if not self.not_use_best_mse:
             last_loss = best_loss
             best_iter = last_best_iter
-        dump_info = (
-            f"quantized {len(quantized_layer_names)}/{(len(quantized_layer_names) + len(unquantized_layer_names))} "
-            f"layers in the block, loss iter 0: {init_loss:.6f} -> iter {best_iter}: {last_loss:.6f}"
-        )
+        if self.iters > 0:
+            dump_info = (
+                f"quantized {len(quantized_layer_names)}/{(len(quantized_layer_names) + len(unquantized_layer_names))} "
+                f"layers in the block, loss iter 0: {init_loss:.6f} -> iter {best_iter}: {last_loss:.6f}"
+            )
+        else:
```

Comment:
Is it possible to use `_quantize_via_rtn_blockwise` for iters=0?

Reply:
_quantize_block is better.
LLMC quantizes model per decoding layer where there are more than 1 Linears. If we need to quantize activation like static FP8, _quantize_via_rtn_blockwise will cache data in itself, which will fail since the model in _quantize_via_rtn_blockwise is actually a block.

Reply:
Have tried _quantize_via_rtn_blockwise locally. During calibration, AR cached ['positional_inputs', 'input_ids', 'attention_mask'] inputs which is the model input rather than block/layer's layer and it failed.